extensions/amp-twitter/0.1/amp-twitter.js

```diff
@@ -68,11 +68,21 @@ class AmpTwitter extends AMP.BaseElement {
       this./*OK*/changeHeight(data['height']);
     }, /* opt_is3P */true);
     listenFor(iframe, 'no-content', () => {
-      if (this.getFallback()) {
+      const fallback = this.getFallback();
+      if (fallback) {
+        // If there is no content, but a fallback is provided.
         this.togglePlaceholder(false);
         this.toggleFallback(true);
+        this./*OK*/changeHeight(fallback.offsetHeight);
```

Comment:
needs to become `fallback/*OK*/.offsetHeight` or presubmit will complain.

Reply:
Done in latest commit.